templates/support/plans-and-pricing.html

```diff
@@ -244,7 +244,7 @@ <h3 class="row-pricing__title">Ubuntu Advantage Desktop</h3>
     <div class="strip-inner-wrapper">
         <div class="ten-col">
             <h2>Landscape on-premises server</h2>
-            <p>If you&rsquo;d prefer a server on premises running behind your firewall, Landscape can be installed on private hardware. This is functionally identical to the SaaS, but you control the hardware.</p>
+            <p>If you&rsquo;d prefer a server on premises running behind your firewall, Landscape can be installed on private hardware. This is functionally identical to the SaaS, but you control the hardware. Requires all the desktop, servers or nodes it manages to have either a Ubuntu Advantage or Landscape seat.</p>
```

Comment:
all good catches... I think the first one is fine... I updated the other two per your suggestions.